router/router.go

```diff
@@ -236,6 +238,12 @@ func (router *Router) handleHTTPEvent(event *eventpkg.Event, w http.ResponseWrit
 	handler := func(w http.ResponseWriter, r *http.Request) {
 		httpdata := event.Data.(*eventpkg.HTTPEvent)
 		httpdata.Params = params
+		if contentType, ok := httpdata.Headers["Content-Type"]; ok &&
+			(strings.HasPrefix(contentType, mimeFormMultipart) || contentType == mimeFormURLEncoded) {
+			if byteSlice, ok := httpdata.Body.([]byte); ok {
+				httpdata.Body = string(byteSlice)
```

Comment:
@elsteelbrain I think this is generally the right approach -- stringifying the request Body -- but I don't know that this is the right place to do it. It would only fix it for `http` events, rather than all events.

[This method](https://github.com/serverless/event-gateway/blob/2ddbcf9723a8878d7e32a0b73f2d94b2396d752b/router/event.go#L46) might be better. cc @mthenw

Reply:
Yup, you are right @alexdebrie. We discussed it earlier today.